content/applications/inventory_and_mrp/maintenance/maintenance_setup.rst

```diff
@@ -0,0 +1,239 @@
+Tools`, and click :guilabel:`New`. This opens a blank equipment form.
+
+In the :guilabel:`Name` field, assign a name for the new equipment. In the :guilabel:`Equipment
+Category` field, click the drop-down menu and select which category this new equipment should belong
+to.
+
+If in a multi-company environment, click the drop-down menu in the :guilabel:`Company` field, and
+select the company in the database to whom the new equipment belongs.
+
+Next to the :guilabel:`Used By` option, select from one of three radio button options:
+:guilabel:`Department`, :guilabel:`Employee`, or :guilabel:`Other`.
+
+.. image:: maintenance_setup/maintenance-setup-new-equipment-left-side.png
+   :align: center
+   :alt: Left-hand side of information fields on new equipment form.
+
+If :guilabel:`Department` is selected, a :guilabel:`Department` field appears below the
+:guilabel:`Used By` option. Click the drop-down menu and select the department that uses this
+equipment.
+
+If :guilabel:`Employee` is selected, an :guilabel:`Employee` field appears below the :guilabel:`Used
+By` option. Click the drop-down menu and select the employee who uses this equipment.
+
+If the :guilabel:`Other` option is selected, both the :guilabel:`Department` and
+:guilabel:`Employee` fields appear below the :guilabel:`Used By` option. Click the drop-down menus
+for the respective fields, and choose which department and employee uses this equipment.
+
+In the :guilabel:`Maintenance Team` field, select the team responsible for this equipment. In the
+:guilabel:`Technician` field, select the team member/user responsible for this equipment.
+
+.. image:: maintenance_setup/maintenance-setup-new-equipment-right-side.png
+   :align: center
+   :alt: Right-hand side of information fields on new equipment form.
+
+In the :guilabel:`Used in location` field, enter the location wherein this equipment will be used,
+if not in an internal work center (e.g. in an office).
+
+In the :guilabel:`Work Center` field, click the drop-down menu, and select which work center this
+equipment will be used in.
+
+Product information
+~~~~~~~~~~~~~~~~~~~
+
+To add any relevant information while creating a new piece of equipment, from the equipment form,
+click the :guilabel:`Product Information` tab.
+
+.. image:: maintenance_setup/maintenance-setup-product-information.png
+   :align: center
+   :alt: Product Information tab with available fields below it.
+
+In the :guilabel:`Vendor` field, add the vendor from which the equipment was purchased. In the
+:guilabel:`Vendor Reference` field, add the product reference number obtained from the vendor, if
+applicable.
+
+In the :guilabel:`Model` field, specify which model this equipment is, if applicable. If the
+equipment is serialized, add a serial number in the :guilabel:`Serial Number` field.
+
+In the :guilabel:`Effective Date` field, click the date to reveal a calendar popover, and select a
```

Comment:
```suggestion
In the :guilabel:`Effective Date` field, click the date to reveal a calendar pop-over window, and select a
```

Reply:
Not implementing per earlier related comment